psi4/driver/p4util/python_helpers.py

```diff
@@ -988,6 +988,17 @@ def _core_wavefunction_set_frequencies(cls, val):
 core.Wavefunction.legacy_frequencies = _core_wavefunction_legacy_frequencies
 core.Wavefunction.set_frequencies = _core_wavefunction_set_frequencies
 
+
+def _core_wavefunction_X(cls):
+    warnings.warn(
+        "Using `psi4.core.Wavefunction.X` instead of `psi4.core.Wavefunction.lagrangian` is deprecated, and in 1.4 it will stop working\n",
```

Comment:
```suggestion
        "Using `psi4.core.Wavefunction.X` instead of `psi4.core.Wavefunction.lagrangian` is deprecated, and in 1.5 it will stop working\n",
```